app/models/item.rb

```diff
@@ -20,9 +20,9 @@ def part_of?(category)
 
   def self.search(search)
     if search
-      where('title LIKE ?', "%#{search}%") + where('description LIKE ?', "%#{search}%")
+      active.where('title LIKE ?', "%#{search}%") + where('description LIKE ?', "%#{search}%")
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.